projects/wave-core-new/src/lib/util/pipes/breakpoint-to-css-string.pipe.ts

```diff
@@ -1,10 +1,10 @@
 import {Pipe, PipeTransform} from '@angular/core';
-import {ColorBreakpointDict} from '../../colors/color-breakpoint.model';
+import {BreakpointDict} from '../../backend/backend.model';
 import {Color} from '../../colors/color';
 
 @Pipe({name: 'breakpointToCssStringPipe'})
 export class BreakpointToCssStringPipe implements PipeTransform {
-    transform(br: ColorBreakpointDict): string {
-        return Color.rgbaToCssString(br.rgba);
+    transform(br: BreakpointDict): string {
+        return Color.rgbaToCssString(br.color);
```

Comment:
Why is this not a component but a pipe?

Reply:
Not sure, I just used the existing functionality.